Dockerfile

```diff
@@ -4,8 +4,7 @@ WORKDIR /app/
 COPY . /app/
 
 FROM base as install-poetry
-ENV POETRY_VIRTUALENVS_CREATE=false \
-    POETRY_VERSION=1.2.0b3
```

Comment:
Rather than remove should just hardcode to 1.2.0 (wanted to remove the need to use an unstable version of 1.2.0

Reply:
fixed